style.css

```diff
@@ -301,7 +284,112 @@ body {
   margin: 0 16px 0 0;
 }
 
+.list ul {
+  display: flex;
+}
+
+.techno li {
+  padding: 5px;
+  background-color: rgb(232, 237, 237);
+  margin-right: 10px;
+  border-radius: 5px;
+}
+
+.list ul li {
+  margin-left: 30px;
+  color: #7a869a;
+}
+
+.techno {
+  display: flex;
+  list-style: none;
+  margin: 10px 0;
+  color: #396df2;
+  height: 2rem;
+}
+
+.sourceBtn:hover,
+.liveBtn:hover {
+  box-shadow: 0 0 4px #003cd3;
+}
+
 @media screen and (min-width: 375px) {
+  .detail {
+    display: none;
+    position: fixed;
+    top: 48%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 90%;
+    height: auto;
+    padding: 20px;
+    box-shadow: 0 1px 10px #344;
+    background: #fff;
+    border-radius: 10px;
+    overflow-y: auto;
+    font-family: 'Poppins', sans-serif;
+    z-index: 20;
+  }
+
+  .cross {
+    position: absolute;
+    right: 1rem;
+    top: 1rem;
+    font-size: 1.5rem;
+    font-weight: bold;
+    color: #344563;
+    cursor: pointer;
+  }
+
+  .status {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .status p {
+    color: #344;
+  }
+
+  .detail img {
+    object-fit: fill;
+    width: 100%;
+  }
+
+  .detail .list {
+    display: flex;
+    margin: 10px 0;
+    font-size: 0.9rem;
+  }
+
+  .switch {
+    display: flex;
+    align-items: center;
+  }
+
+  .switch img {
+    width: 1rem;
+    margin: 0 5px;
+  }
+
+  .liveBtn {
+    padding: 8px;
+    border-radius: 5px;
+    border: 1px solid #6070ff;
+    color: #396df2;
+    display: flex;
+    margin-right: 10px;
+    cursor: pointer;
+  }
+
+  .sourceBtn {
+    padding: 8px 12px;
+    border-radius: 5px;
+    border: 1px solid #6070ff;
+    color: #396df2;
+    display: flex;
+    cursor: pointer;
+  }
+
   .navBar {
```

Comment:
- [ Optional ] In your mobile version of humburger menu design is not align the `Figma` template design and when I click the navbar link `portfolio` , `about` and `contact` it is working but the background blur is always showed when I click each  links actually it should be disappear when I click associated link and check the below the picture.

**Your Design**

![Screenshot (204)](https://github.com/mioOleen/Portfolio-1/assets/118582316/c3a9e4da-1c1c-4ba5-b2b7-5e00fa3918d3)


**Figma Design**

![Screenshot (205)](https://github.com/mioOleen/Portfolio-1/assets/118582316/77332e29-6ce9-4c2e-8e7d-c3bb48e51fc7)